Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj

```diff
@@ -0,0 +1,149 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You are referencing dlls from TrouterBasedEventChannel project here. You don't need to add those dlls to MessagingAfterJoinMeeting directory.